src/woo_publications/publications/models.py

```diff
@@ -33,6 +33,7 @@
 
 
 class Publication(models.Model):
+    id: int  # implicitly provided by django
```

Comment:
I updated the type in the factory, so now pyright knows that `DocumentFactory.create()` returns a `Document`, and existing tests have assertions using `Document.id` which pyright didn't understand because it's Django magic.

Reply:
The updated factory generic type: https://github.com/GeneriekPublicatiePlatformWoo/registratie-component/pull/135/files#diff-ea16fd5a8434420cfe3ea0f8206ab5ab06d7d30aa556c2226741744572d3bd60R33